inc/admin/dashboard/namespace.php

```diff
@@ -154,6 +185,44 @@ function lowly_user() {
 	);
 }
 
+/**
+ * Callback for /wp-admin and /wp-admin/user widget
+ *
+ * Renders book invitations if user has at least one pending book invitation.
+ */
+function pending_invitations_callback() {
+	global $wpdb;
+
+	$current_blog_id = get_current_blog_id();
+	$user_id = get_current_user_id();
+
+	$invitations = $wpdb->get_results( $wpdb->prepare( "SELECT meta_key, meta_value FROM $wpdb->usermeta WHERE meta_key LIKE %s AND user_id = %d", 'new_user_%', $user_id ) );
```

Comment:
maybe we could use: https://developer.wordpress.org/reference/functions/get_user_meta/, what do you think?

Reply:
Not sure if we can use get_user_meta to perform a `LIKE` operation. Happy to discuss during review metting